docs/DeveloperGuide.adoc

```diff
@@ -14,7 +14,9 @@ ifdef::env-github[]
 endif::[]
 :repoURL: https://github.com/se-edu/addressbook-level3/tree/master
 
-By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
+image::title.png[width="600"]
+
+By: `Team CS2103-F09-3`      Since: `Sep 2019`      Licence: `MIT`
 
 == Setting up
```

Comment:
You have not changed the Design section at all, even in the diagrams. Are you sure that the structure of your application can be mapped so perfectly onto the base Address Book application?